modules/microshift-rhoai-supported-mserv-runtimes.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-rhoai-supported-models_{context}"]
```

Comment:
Maybe we should link RHOAI docs about these runtimes?

Reply:
We do, I just can't link in the module.